src/Umbraco.Forms.Integrations.Crm.Hubspot/Services/HubspotContactService.cs

```diff
@@ -21,14 +22,14 @@ namespace Umbraco.Forms.Integrations.Crm.Hubspot.Services
 {
     public class HubspotContactService : IContactService
     {
-        // Using a static HttpClient (see: https://www.aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/).
+        //Using a static HttpClient(see: https://www.aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/).
```

Comment:
I think we can improve this pattern now we aren't have to multi-target.  Have a read/look into using IHttpClientFactory [here](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests) and [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-6.0)

I think we probably want to register a named or typed HttpClient and use that method.  There's an example of using a named one the marketplace solution (registering one [here](https://github.com/umbraco/Umbraco.Marketplace/blob/main/src/PackagesRepository/PackagesRepository.Infrastructure/Extensions/ServiceCollectionExtension.cs#L72), and using it [here](https://github.com/umbraco/Umbraco.Marketplace/blob/main/src/PackagesRepository/PackagesRepository.Infrastructure/Components/NuGetSync/NuGetPackageDataRetriever.cs#L52)).